### ISSUE TITLE
Awesome! Updated Droidplay fork?! Haven't tried yet. Excited. F-Droid or IzzyonDroid Repo Plans?!

### ISSUE THREAD

**Issue by lpstandardrp** [warren-bank/Android-AirPlay-Client#4]
TL/DR: **You plan to add to F-Droid or IzzyonDroid Repo?!**

Awesome! 

Updated Droidplay fork?! 
Downloaded, Haven't tried yet. Excited.

First, I want to throw this out there in case it may benefit someone (or many persons, or the community(ies)..., or change or save the future of all living present and future kind) 
**You plan to add to F-Droid or IzzyonDroid Repo?!**

I honestly stumbled upon this by luck, *after* looking for a Droidplay updated fork (for some time, numerous search routes- Github-Forks, AlternativeTo, LibHunt, ...maybe more)... so Just saying it may benefit enough to warrant the submission process to either repo... 
I do not know. And at the same time, I would be a chump for not mentioning it :} <3

P.s. The website host for IzzyonDroid Repo / dev(? - I think; I dont know)/ Github user (his UserName slips my mind, probably @ IzzyonDroid), has been very helpful in guiding devs of apps on Github through the F-Droid | IzzyOnDroid Repo submission proces,  IIRC.

**Comment by Poussinou** [comment]
It's already on IzzyOnDroid [here](https://apt.izzysoft.de/fdroid/index/apk/com.github.warren_bank.airplay_client).
